example/kotlinlib/basic/1-simple/foo/test/src/foo/FooTest.kt

```diff
@@ -1,15 +1,11 @@
 package foo
 
-import foo.generateHtml
 import io.kotest.core.spec.style.FunSpec
 import io.kotest.matchers.shouldBe
 
-class FooTest : FunSpec({
-    test("testSimple") {
-        generateHtml("hello") shouldBe "<h1>hello</h1>"
-    }
+class FooTest :
```

Comment:
this doesn't look like a reasonable change, but I don't know `ktfmt` default rules. Overall in the Kotlin system `ktlint` is much more popular than `ktfmt`.

Reply:
This is just the default `ktfmt` as there's no configuration. I'm guessing it's better to any formatter in place and we can let which the formatter battle to happen in parallel since it should then be a fairly easy switch.

Reply:
Then why not use `ktlint` (as it is proposed in https://github.com/com-lihaoyi/mill/issues/3829) provided by `KtlintModule`? It will require some changes to the run in the module-independent manner, but it will be easy to do by looking on `KtfmtModule`.

Reply:
Oh yeah, it is `ktlint` in the issue. I misread that. Forgive me, I am actually dyslexic.

Reply:
No worries, let's try `ktlint` and see how it looks. Then we can pick the one that looks better.

I noticed `KtlintModule` does not have the companion `object extends External Module` that the other autoformatters do, making it more annoying to use ad-hoc. We should probably add such an companion external module. `KtfmtModule`, `ScalafmtModule`, and `PalantirFormatModule` can be used as a reference as to how it is done